templates/kubernetes/index.html

```diff
@@ -47,15 +47,15 @@ <h2>Cloud Native Platform</h3>
   </div>
 </section>
 
-<section id="partnerships-google-gke" class="p-strip is-bordered">
-  <div class="row u-vertically-center">
-    <div class="col-4  u-hide--small u-align--center">
-      <img src="{{ ASSET_SERVER_URL }}ba70dd8a-gce.png?h=160" alt="Google Container Engine logo" />
+<section class="p-strip is-bordered">
+  <div class="row u-equal-height u-vertically-center">
+    <div class="col-5 suffix-1 u-align--center u-hide--small u-align--center u-hide--small">
+      <img src="{{ ASSET_SERVER_URL }}ba70dd8a-gce.png?w=380" alt="Google Container Engine" />
     </div>
-    <div class="col-8">
-      <h2>Google&nbsp;GKE</h2>
-      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes &reg;. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s&nbsp;alignment.</p>
-      <p><a href="https://cloud.google.com/kubernetes-engine/docs/node-images#ubuntu" class="p-link--external">Try GKE with Ubuntu now</a></p>
+    <div class="col-7">
+      <h2>In partnership with Google GKE</h2>
+      <p>Google and Canonical together enable smooth hybrid operations between Google&rsquo;s Container Engine (GKE) service with Ubuntu worker nodes, and Canonical&rsquo;s Distribution of Kubernetes&reg;<a href="#fn1" id="r1">*</a>. Choose Canonical&rsquo;s Kubernetes to be sure your container workloads can migrate to GKE thanks to our kernel-to-k8s alignment.</p>
+      <p><a href="https://cloud.google.com/container-engine/docs/node-images#ubuntu_beta" class="p-link--external">Try GKE with Ubuntu now</a></p>
```

Comment:
The link is:
https://cloud.google.com/kubernetes-engine/docs/node-images#ubuntu

Reply:
Updated... good catch